docker-compose.yml

```diff
@@ -30,7 +28,7 @@ services:
       - ./:/usr/src/stunning-journey-api
       - /usr/src/stunning-journey-api/node_modules
     ports:
-      - '5000:5000'
+      - '5001:5000'
```

Comment:
```suggestion
      - '5001:5001'
```

Reply:
In fact, the idea is to have different ports because then it would be easy to debug in Kubernetes. If everything has the same port, it would be a mess